Command/RemoveThumbsCommand.php

```diff
@@ -182,13 +177,14 @@ public function getContext()
     public function getFormat(MediaProviderInterface $provider, $context)
     {
         $format = $this->input->getArgument('format');
+
         if (null === $format) {
             $formats = array_keys($provider->getFormats());
             $formats[] = '<ALL THUMBNAILS>';
 
-            $dialog = $this->getHelperSet()->get('dialog');
-            $formatKey = $dialog->select($this->output, 'Please select the format', $formats);
-            $format = $formats[$formatKey];
+            $question = new ChoiceQuestion('Please select the format', $formats);
```

Comment:
This variable seems to be use only one time.

It should not be initialized.

cc @greg0ire

Reply:
Dude...

Reply:
![stahp](https://i.imgur.com/6Ujeof9.gif)

Reply:
@greg0ire Sorry for the copy paste.

Plus, I never remember the rules name, I have to write it somewhere. :trollface:

Reply:
Just bookmark this: https://a-nickels-worth.blogspot.fr/2016/04/a-guide-to-naming-variables.html ;)